docs/1.getting-started/6.data-fetching.md

```diff
@@ -311,7 +311,7 @@ Be very careful before proxying headers to an external API and just include head
 
 #### Example: Pass Cookies From Server-side API Calls on SSR Response
 
- If you want to pass on/proxy cookies in the other direction, from an internal request back to the client, you will need to handle this yourself.
+ If you want to pass on/proxy cookies in the other direction, from an internal request and back to the client, you will need to handle this yourself.
```

Comment:
I'd consider the original form.
```suggestion
 If you want to pass on/proxy cookies in the other direction, from an internal request back to the client, you will need to handle this yourself.
```

Reply:
Hmmm, I don't know what actually sounds better..

Reply:
I don't know, I guess it sounds better? I can't quite explain but it is almost more "continuous" this way. But what do you think? Should we remove the 'and'?

Reply:
Yes please, in this case I would certainly favor the version without _and_.